data/misc/specialists.spec

```diff
@@ -23,18 +24,29 @@ dx = 15
 dy = 20
 
 tiles = { "row", "column", "tag"
-  0,  0, "specialist.elvis_0"
-  0,  1, "specialist.elvis_1"
-  0,  0, "specialist.entertainer_0"
-  0,  1, "specialist.entertainer_1"
-  0,  2, "specialist.scientist_0"
-  0,  3, "specialist.scientist_1"
-  0,  4, "specialist.taxman_0"
-  0,  5, "specialist.taxman_1"
-  0,  6, "specialist.worker_0"
-  0,  7, "specialist.worker_1"
-  0,  8, "specialist.farmer_0"
-  0,  9, "specialist.farmer_1"
-  0, 10, "specialist.merchant_0"
-  0, 11, "specialist.merchant_1"
+; Citizen icons:
+  0, 0, "citizen.happy_0"
```

Comment:
This duplicates stuff from `small.spec`, I'd rather have a single copy

Reply:
Yea I noticed it too
```
[info] freeciv21-client (../client/tilespec.cpp:1060) - Loading tileset "hexemplio".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.happy_0".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.happy_1".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.content_0".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.content_1".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.unhappy_0".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.unhappy_1".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.angry_0".
[critical] freeciv21-client (../client/tilespec.cpp:1405) - warning: already have a sprite for "citizen.angry_1".
```